### PR TITLE
Updates for Export/Import doc

### PR DESCRIPTION
**Pull request by mwichmann**
Mention mutable objects - if an Sconscript imports a previously exported variable, it is the same object, so changes propagate.  Some wording tweaks in SConscript function.

Doc-only change.

## Contributor Checklist:

* [ ] I have created a new test or updated the unit tests to cover the new/changed functionality.
* [X] I have updated `CHANGES.txt` (and read the `README.rst`)
* [X] I have updated the appropriate documentation

### CODE REVIEW

File: SCons/Script/SConscript.xml [hunk 1]
```diff
@@ -161,25 +161,45 @@ is used if no value is specified.
 </arguments>
 <summary>
 <para>
-Exports variables from the current
-SConscript file to a global collection where they can be
-imported by other SConscript files.
+Exports variables for sharing with other SConscript files.
+The variables are added to a global collection where
+they can be imported by another SConscript file.
```
Comment (bdbaddog) [inline SCons/Script/SConscript.xml]: I think other file(s) is more accurate here?

File: SCons/Script/SConscript.xml [hunk 2]
```diff
@@ -372,7 +400,7 @@ Return('val1 val2')
 
 <scons_function name="SConscript">
 <arguments>
-(scripts, [exports, variant_dir, duplicate, must_exist])
+(scriptname, [exports, variant_dir, duplicate, must_exist])
```
Comment (bdbaddog) [inline SCons/Script/SConscript.xml]: scriptnames? It can be plural right?

Comment (bdbaddog) [reply SCons/Script/SConscript.xml]: the actual arg is `*ls` ?

Comment (mwichmann) [reply SCons/Script/SConscript.xml]: was trying to match up with wording. as noted elsewhere, unnamed args aren't really helpful in doc generation etc. but we have a good number spread around. SConscript is a weird function, can take 0, 1 or 2 positional args. the 0 part being a bit odd...

Comment (mwichmann) [reply SCons/Script/SConscript.xml]: and yes, pluralizing it is ok, if you prefer that.

Comment (bdbaddog) [reply SCons/Script/SConscript.xml]: I think pluralizing would be better. though I think rarely used, it is accurate.

**Pull request merged**